FluentUI/build.gradle

```diff
@@ -9,7 +9,7 @@ apply from: '../config.gradle'
 apply from: '../publish.gradle'
 
 android {
-    compileSdkVersion constants.compileSdkVersion
+    compileSdk 31
```

Comment:
Intended change? The variable seems to be holding 31 value only

Reply:
Yes, it was intended change.

Reply:
Any reason for the change from "compileSdk" to "compileSdkVersion"? As far as I can see both are the same.

Also, use the same constant instead of hardcoded value

Reply:
I checked and restored the previous line. There was no need for change. Please check and resolve the conversation.